project.clj

```diff
@@ -43,6 +43,7 @@
                  [prismatic/schema ~schema-version]
                  [metosin/schema-tools ~schema-tools-version]
                  [threatgrid/ctim "0.4.9"]
+                 [threatgrid/clj-momo "0.2.6-SNAPSHOT"]
```

Comment:
This should come from CTIM.  Release an updated CTIM once the clj-momo changes are stable.